### PR TITLE
BNGP-3538 content changes

### PR DESCRIPTION
**Pull request by AjinkyaJawalkar**

### CODE REVIEW

**Review by KiranVarma02 — COMMENTED** [review]
LGTM. I am happy to approve if you can show me how to get to credit changes in local.

**Comment by AjinkyaJawalkar** [comment]
> LGTM. I am happy to approve if you can show me how to get to credit changes in local.

@KiranVarma02 You just need to take the latest branch on your local and run the application. you can directly navigate to the respective pages like _http://localhost:3000/credits-estimation/credits-tier_ and on press of continue you will be on _http://localhost:3000/credits-estimation/credits-cost_

File: packages/webapp/src/routes/credits-estimation/credits-tier.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import constants from '../../credits/constants.js'
 import calculateCost from '../../credits/calculate.js'
 import Joi from 'joi'
 
-const errorMessage = { text: 'Enter the credit unit from the metric. The credit unit must be entered up to 2 decimal places, like 23.75.' }
+const errorMessage = { text: 'Enter the number of credits from the metric up to 2 decimal places, like 23.75.' }
```
Comment (paulsimonandrews) [inline packages/webapp/src/routes/credits-estimation/credits-tier.js:5]: Update `metric` to `Metric`

File: packages/webapp/src/routes/__tests__/credits-estimation/credits-tier.spec.js [hunk 2]
```diff
@@ -65,7 +65,7 @@ describe(url, () => {
       postOptions.payload = badPayload
       const res = await submitPostRequest(postOptions, 200)
       expect(res.result).toContain('There is a problem')
-      expect(res.result).toContain('Enter the credit unit from the metric. The credit unit must be entered up to 2 decimal places, like 23.75.')
+      expect(res.result).toContain('Enter the number of credits from the metric up to 2 decimal places, like 23.75.')
```
Comment (paulsimonandrews) [inline packages/webapp/src/routes/__tests__/credits-estimation/credits-tier.spec.js:68]: Update `metric` to `Metric`

**Review by paulsimonandrews — CHANGES_REQUESTED** [review]
Just need to update `metric` to `Metric` in the error message

**Review by paulsimonandrews — APPROVED** [review]

**Comment by shaundefra** [comment]
tested by leena

**Pull request merged**